### PR TITLE
media-libs/pulseaudio-qt6: Add Qt6 support

### PR DESCRIPTION
**Pull request by aidanharris**

### CODE REVIEW

File: media-libs/pulseaudio-qt6/pulseaudio-qt6-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+if [[ ${KDE_BUILD_TYPE} = release ]]; then
+	SRC_URI="mirror://kde/stable/${MY_PN}/${MY_P}.tar.xz"
+	KEYWORDS="~amd64 ~arm64 ~ppc64 ~riscv ~x86"
+fi
+
+LICENSE="LGPL-2.1"
+SLOT="0/3"
+
+RDEPEND="
+	>=dev-qt/qtgui-${QTMIN}:6
```
Comment (thesamesam) [inline media-libs/pulseaudio-qt6/pulseaudio-qt6-9999.ebuild:29]: See https://invent.kde.org/libraries/pulseaudio-qt/-/blob/master/CMakeLists.txt?ref_type=heads#L40 vs https://wiki.gentoo.org/wiki/Project:Qt/Qt6_migration_notes.

Comment (aidanharris) [reply media-libs/pulseaudio-qt6/pulseaudio-qt6-9999.ebuild:29]: Thanks. I didn't know that page existed.

Comment (thesamesam) [reply media-libs/pulseaudio-qt6/pulseaudio-qt6-9999.ebuild:29]: It's new :D

**Comment by a17r** [comment]
kde-misc/kdeconnect still is the only revdep of media-libs/pulseaudio-qt. That means a new package name just for it being ported to Qt6 is not justifiable. Let's pin down kdeconnect:5 to <media-libs/pulseaudio-qt-1.4 instead. But we do not even have a release of that yet, as kdeconnect is not yet officially shipped as Qt6-based in 24.02:

https://community.kde.org/KDE_Gear/24.02_Release_notes#Tarballs_have_switched_from_being_Qt5_based_to_Qt6-based

That means we are not going to switch kdeconnect to KF6 yet, either. I'm going to ask upstream about it tomorrow though.

File: media-libs/pulseaudio-qt6/pulseaudio-qt6-9999.ebuild (hunk 1 above)
Comment (a17r) [reply media-libs/pulseaudio-qt6/pulseaudio-qt6-9999.ebuild]: Regardless, you would have noticed upon pkgcheck, which is a bare minimum before filing a PR.

**Comment by jospezial** [comment]
Next 24.xx tag will have it.
https://invent.kde.org/network/kdeconnect-kde/-/commit/7f03aa548c233ab44d1cbb12f3ced8303ce95b9c
"Fully port to Qt6"

https://invent.kde.org/network/kdeconnect-kde/-/merge_requests/611
was merged also.

**Comment by a17r** [comment]
very good.

**Pull request closed without merging**